app/build.gradle

```diff
@@ -90,6 +94,7 @@ dependencies {
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0'
+    implementation 'androidx.security:security-crypto:1.1.0-alpha01'
```

Comment:
Can you please add a TODO to the README to elaborate on the risks on API 21-22? Just a quick note to elaborate on it when we write the documentation.

Reply:
Never mind, I added a TODO to https://github.com/relaycorp/relaynet-gateway-android/issues/1